Session/Settings/SettingsViewModel.swift

```diff
@@ -285,18 +319,16 @@ class SettingsViewModel: SessionTableViewModel, NavigationItemSource, Navigatabl
                     title: SessionCell.TextInfo(
                         state.profile.displayName(),
                         font: .titleLarge,
-                        alignment: .center
-                    ),
-                    trailingAccessory: .icon(
-                        .pencil,
-                        size: .small,
-                        customTint: .textSecondary
+                        alignment: .center,
+                        trailingImage: (state.isSessionPro ?
+                            ("ProBadge", SessionProBadge(size: .medium).toImage()) :
```

Comment:
Just wondering whether we should add a function to `ImageDataManager` which takes a cache key and a closure that returns an image, `ImageDataManager` can then return a previously cached version of the image (that way we only need to run `SessionProBadge.toImage()` once for each size so it'd have less of an impact on UI performance)

If we end up reworking the `@You` behaviour to use a similar pattern then it could benefit from the same mechanism